src/Sulu/Bundle/MediaBundle/Media/ImageConverter/ImagineImageConverter.php

```diff
@@ -124,8 +124,14 @@ public function getSupportedOutputImageFormats(?string $mimeType): array
 
                 switch ($mimeType) {
                     case 'image/png':
+                        $preferredExtension = 'png';
+                        break;
                     case 'image/svg+xml':
+                    case 'image/svg':
                         $preferredExtension = 'png';
+                        if (class_exists('Contao\ImagineSvg\Imagine')) {
```

Comment:
Would import it for better IDE refractoring support:

`use Contao\ImagineSvg\Imagine as SvgImagine;`